### PR TITLE
Users/bilalmehrban/retrieve all calendar entry attachments

### PR DESCRIPTION
**Pull request by bilalmehrban**
image

**Followed the Pure coding format. If there is any thing that still needs to be worked on please let me know I will update accordingly.**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Logic.cs [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 namespace OtripleS.Web.Api.Tests.Unit.Services.CalendarEntryAttachments
 {
+    using System;
+    using System.Linq;
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Logic.cs:15]: Why are you putting the usings on the inside of your namespace scope?

File: OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Validations.RetrieveAll.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        [Fact]
+        public void ShouldLogWarningOnRetrieveAllWhenCalendarEntryAttachmentsWereEmptyAndLogIt()
+        {
+            // given
+            IQueryable<CalendarEntryAttachment> emptyStorageCalendarEntryAttachments = new List<CalendarEntryAttachment>().AsQueryable();
+            IQueryable<CalendarEntryAttachment> expectedCalendarEntryAttachments = emptyStorageCalendarEntryAttachments;
+
+            this.storageBrokerMock.Setup(broker =>
+                    broker.SelectAllCalendarEntryAttachments())
+                .Returns(emptyStorageCalendarEntryAttachments);
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Validations.RetrieveAll.cs:28]: Please follow our indentation standard:

```csharp
SomeMethodCall()
    .SomeChainCall()
        .SomeOtherCall();
```

File: OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.cs [hunk 3]
```diff
@@ -60,5 +66,9 @@ private static Filler<CalendarEntryAttachment> CreateCalendarEntryAttachmentFill
 
         private static SqlException GetSqlException() =>
           (SqlException)FormatterServices.GetUninitializedObject(typeof(SqlException));
+
+        private static IQueryable<CalendarEntryAttachment> CreateRandomCalendarEntries(DateTimeOffset dateTime) =>
+            CreateCalendarEntryAttachmentFiller(dateTime)
+                .Create(GetRandomNumber()).AsQueryable();
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.cs:72]: You can move this line above by 1

File: OtripleS.Web.Api/Services/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs [hunk 4]
```diff
@@ -11,10 +11,12 @@
...
 namespace OtripleS.Web.Api.Services.CalendarEntryAttachments
 {
+    using System.Linq;
+
     public partial class CalendarEntryAttachmentService
     {
         private delegate ValueTask<CalendarEntryAttachment> ReturningCalendarEntryAttachmentFunction();
-
+        private delegate IQueryable<CalendarEntryAttachment> ReturningCalendarEntryAttachmentsFunction();
         private async ValueTask<CalendarEntryAttachment> TryCatch(
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Services/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs:20]: Separate the two-liner code blocks by a new empty line from everything before and after

File: OtripleS.Web.Api/Services/CalendarEntryAttachments/ICalendarEntryAttachmentService.cs [hunk 5]
```diff
@@ -6,12 +6,15 @@
...
 using System;
 using System.Threading.Tasks;
+using System.Linq;
 
 namespace OtripleS.Web.Api.Services.CalendarEntryAttachments
 {
     public interface ICalendarEntryAttachmentService
     {
         ValueTask<CalendarEntryAttachment> RetrieveCalendarEntryAttachmentByIdAsync
             (Guid calendarEntryId, Guid attachmentId);
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Services/CalendarEntryAttachments/ICalendarEntryAttachmentService.cs:16]: Break these two variables with a new line each

File: OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Logic.cs (hunk 1 above)
Comment (bilalmehrban) [reply OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Logic.cs:15]: I have installed code ReSharper and I think it is doing that.

File: OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Validations.RetrieveAll.cs (hunk 2 above)
Comment (bilalmehrban) [reply OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Validations.RetrieveAll.cs:28]: Should I update and send a PR again or will it be taken care in the code rub?

Comment (hassanhabib) [reply OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Validations.RetrieveAll.cs:28]: You should fix it in this PR.

**Comment by hassanhabib** [comment]
@bilalmehrban make sure you fix conflicts

**Pull request closed without merging**